anta/tests/cvx.py

```diff
@@ -87,3 +87,75 @@ def test(self) -> None:
         cluster_status = command_output["clusterStatus"]
         if (cluster_state := cluster_status.get("enabled")) != self.inputs.enabled:
             self.result.is_failure(f"Management CVX status is not valid: {cluster_state}")
+
+
+class VerifyMcsServerMounts(AntaTest):
+    """Verify if all MCS server mounts are in mountStateMountComplete.
+
+    Expected Results
+    ----------------
+    * Success: The test will pass if the MCS mount status on MCS server are mountStateMountComplete.
+    * Failure: The test will fail even if one switch's MCS server mount status is not  mountStateMountComplete.
+
+    Examples
+    --------
+    ```yaml
+    anta.tests.cvx:
+    - VerifyMcsServerMounts:
+        expected_connection_count: 100
```

Comment:
wrong example

```suggestion
    anta.tests.cvx:
    - VerifyMcsServerMounts:
```

Reply:
I am using expected_connection_count here as well. Because there can be 100s of connections and I do want to validate if anything is missing as part of show cvx mounts.